packages/rest/src/__tests__/unit/rest.server/rest.server.unit.ts

```diff
@@ -123,6 +124,18 @@ describe('RestServer', () => {
         /Base path cannot be set as the request handler has been created/,
       );
     });
+
+    it('honors requestBodyParser options in config', async () => {
+      const parserOptions: RequestBodyParserOptions = {json: {limit: '1mb'}};
+      const app = new Application({
+        rest: {requestBodyParser: parserOptions},
+      });
+      app.component(RestComponent);
+      const server = await app.getServer(RestServer);
+      expect(server.getSync(RestBindings.REQUEST_BODY_PARSER_OPTIONS)).to.equal(
```

Comment:
nitpick: can we use `get` here instead of `getSync` with `await` to keep it in sync with the async/await style of the test?

Reply:
I wanted to assert that the `options` can be resolved synchronously.